app/src/main/res/layout/request_desktop_check_menu_item.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="48dp"
+    android:background="?android:attr/selectableItemBackground"
+    android:clickable="true"
+    android:focusable="true"
+    android:orientation="horizontal"
+    android:paddingStart="16dp"
+    android:paddingEnd="0dp">
+
+    <TextView
+        style="@android:style/TextAppearance.Material.Menu"
+        android:layout_width="0dp"
+        android:layout_height="48dp"
+        android:layout_weight="1"
+        android:clickable="false"
+        android:ellipsize="end"
+        android:focusable="false"
+        android:gravity="center_vertical"
+        android:lines="1"
+        android:text="@string/preference_performance_request_desktop_site"
+        android:textSize="16sp" />
+
+    <CheckBox
+        android:id="@+id/check_menu_item_checkbox"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_vertical|end"
+        android:checked="true"
+        android:clickable="true"
+        android:ellipsize="end"
+        android:focusable="true"
+        android:gravity="center"
+        android:paddingEnd="8dp" />
```

Comment:
We should set the text on the checkbox.

* Otherwise screenreaders won't be able to announce what this checkbox is for.
* Clicking on the text will automatically toggle the checkbox

I'm not sure whether we can change the orientation of the checkbox though. I think by default it is (checkbox - text).